tests/sles4sap/saptune/mr_test_run.pm

```diff
@@ -482,24 +485,50 @@ sub wrapup_log_file {
 sub wrap_script_run {
     my ($self, $args) = @_;
     my $ret = '';
+    my $ret_tmp = '';
     my $log = '/tmp/log';
 
     $ret = script_run "$args > $log 2>&1";
     if ($ret) {
-        script_run "cat $log";
-        record_soft_failure('Error found:jsc#TEAM-7435');
-        # These are known issues so mark them as 'softfail' not 'fail' for not blocking 'bot' auto approval
-        # - sysstat service [FAIL] exited disabled != dead disabled
-        # - sysstat (cronjob) [FAIL] ok == no link
-        # - UserTasksMax [FAIL] max == 12288
-        # - UserTasksMax (sapconf DropIn) [FAIL] regular == missing
-        if (
-            script_run(
-"grep -Ev '(sysstat service.*FAIL.*exited disabled|sysstat \(cronjob\) .*FAIL.*ok|UserTasksMax.*FAIL.*max|UserTasksMax \(sapconf DropIn\).*FAIL.*regular)' $log"
-            ))
-        {
+        # Debug purpose
+        my $output = script_output("cat $log");
```

Comment:
Should/could be adding something to activate all these new code only if tested saptune is 3.1 and not for older versions?

Reply:
For all saptune versions as here does not check saptune versions and all 12sp5/15sp+ products using saptune3.1 now.
It only shows the output of `mr_test` when the execution of `mr_test` failed.